modules/sovrnBidAdapter.js

```diff
@@ -50,27 +53,48 @@ export const spec = {
         }
         iv = iv || utils.getBidIdParameter('iv', bid.params)
 
-        let bidSizes = (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.banner.sizes) || bid.sizes
-        bidSizes = ((utils.isArray(bidSizes) && utils.isArray(bidSizes[0])) ? bidSizes : [bidSizes])
-        bidSizes = bidSizes.filter(size => utils.isArray(size))
-        const processedSizes = bidSizes.map(size => ({w: parseInt(size[0], 10), h: parseInt(size[1], 10)}))
         const floorInfo = (bid.getFloor && typeof bid.getFloor === 'function') ? bid.getFloor({
           currency: 'USD',
-          mediaType: 'banner',
+          mediaType: bid.mediaTypes && bid.mediaTypes.banner ? 'banner' : 'video',
           size: '*'
         }) : {}
         floorInfo.floor = floorInfo.floor || utils.getBidIdParameter('bidfloor', bid.params)
 
         const imp = {
           adunitcode: bid.adUnitCode,
           id: bid.bidId,
-          banner: {
+          tagid: String(utils.getBidIdParameter('tagid', bid.params)),
+          bidfloor: floorInfo.floor
+        }
+
+        if (bid.mediaTypes && bid.mediaTypes.banner) {
+          let bidSizes = (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.banner.sizes) || bid.sizes
+          bidSizes = ((utils.isArray(bidSizes) && utils.isArray(bidSizes[0])) ? bidSizes : [bidSizes])
+          bidSizes = bidSizes.filter(size => utils.isArray(size))
+          const processedSizes = bidSizes.map(size => ({w: parseInt(size[0], 10), h: parseInt(size[1], 10)}))
+
+          imp.banner = {
             format: processedSizes,
             w: 1,
             h: 1,
-          },
-          tagid: String(utils.getBidIdParameter('tagid', bid.params)),
-          bidfloor: floorInfo.floor
+          };
+        }
+        if (bid.mediaTypes && bid.mediaTypes.video) {
+          let bidSizes = bid.mediaTypes.video.playerSize
+          bidSizes = (utils.isArray(bidSizes) && utils.isArray(bidSizes[0])) ? bidSizes : [bidSizes]
+          bidSizes = bidSizes.filter(size => utils.isArray(size))
+
+          const video = utils.deepAccess(bid, 'mediaTypes.video') || {}
+
+          imp.video = {
```

Comment:
We should support all of the video parameters 
I really like how this adapter did it https://github.com/prebid/Prebid.js/blob/832c13ee4126ac20a81e00fc35c8d71fa0afff25/modules/deepintentBidAdapter.js
specifically 
https://github.com/prebid/Prebid.js/blob/832c13ee4126ac20a81e00fc35c8d71fa0afff25/modules/deepintentBidAdapter.js#L179